tools/set_project_event_subs.php

```diff
@@ -7,12 +7,18 @@
 require_login();
 
 $projectid = validate_projectID('projectid', @$_POST['projectid']);
+$project = new Project($projectid);
 $return_uri = urldecode($_POST['return_uri']);
 
 $subs = array();
 $unsubs = array();
 foreach ( $subscribable_project_events as $event => $label )
 {
+    if (!can_user_subscribe_to_project_event( $pguser, $projectid, $event ))
```

Comment:
@srjfoo raised a valid concern about this. If someone is a PP for a project, subscribes to this event and then returns the project to the pool (stops being the PP), they cannot unsubscribe themselves from the event. 

We could _silently_ unsubscribe them from the event (add `unsubscribe_user_from_project_event` before the `continue`), however I am not sure whether that is the best way to handle it.

Reply:
I think silently unsubscribing them from the event makes sense, but we want that to be in the code that returns the project to the PP pool.

Reply:
I agree -- I just hadn't mentioned it yet. The best time to do it is when the project is returned to the pool.